bin/fuel-core/Cargo.toml

```diff
@@ -40,6 +41,7 @@ test-case = { workspace = true }
 [features]
 debug = ["fuel-core/debug"]
 default = ["debug", "metrics", "relayer", "rocksdb"]
+env = ["dep:dotenvy"]
```

Comment:
Maybe it also should be part of the `production` and `debug` feature

Reply:
I can understand adding it to the `debug` feature. I can definitely add it there.

I was looking at some of the deployment architecture to see if there would be any conflicts here, because I don't know how this will interact with existing environment variables like the ones set in K8s any configs or helm. It may override them. Is there any risk here?

Reply:
If we don't use `.env`, it shouldn't affect the current operator or chart deployment